.github/workflows/ci.yml

```diff
@@ -1,16 +1,17 @@
+---
 name: Build and test astro databricks provider
-on:
+on:  # yamllint disable-line rule:truthy
```

Comment:
@pankajkoti Curious about this comment!

Reply:
YAML linter was complaining that the value needs to be True or False. This is a way we have been using for the YAML linter in GitHub CI workflows across other projects to for disabling this false lint report.